src/lib/battery/battery.h

```diff
@@ -168,11 +164,12 @@ class Battery : public ModuleParams
 	bool _battery_initialized{false};
 	float _voltage_v{0.f};
 	AlphaFilter<float> _voltage_filter_v;
```

Comment:
I think you mentioned it already once to me, but are 
_voltage_filter_v
_current_filter_a
now still required with the other filters you added here?

Reply:
_voltage_filter_v is used to check if the battery is disconnected, replaced it with raw voltage measurement and removed the filter.
_current_filter_v was no longer used anywhere, removed it.

Reply:
perfect